### PR TITLE
Generate random secret key animation

### PR DESCRIPTION
**Pull request by akonior**

### CODE REVIEW

File: zkmarek/video/mobjects/wallet.py [hunk 1]
```diff
@@ -9,22 +11,28 @@ class Wallet(VGroup):
...
+    secret_key: VGroup
+    secret_key_label: Text
+    secret_key_value: Text
     address: Text
 
-    def __init__(self, secret_key = "", address = ""):
+    def __init__(self, secret_key="", address=""):
         super().__init__()
-        self.secret_key = Text(secret_key, font_size=24, font="Monaco", color=SECONDARY_COLOR)
+        self.secret_key_label = Text(secret_key, font_size=24, font="Monaco", color=SECONDARY_COLOR)
```
Comment (marekkirejczyk) [inline zkmarek/video/mobjects/wallet.py:21]: "Monaco" -> PRIMARY_FONT

File: zkmarek/video/mobjects/wallet.py [hunk 2]
```diff
@@ -34,3 +42,15 @@ def animate_in(self, scene):
         scene.play(FadeIn(self.border), FadeIn(self.clip), FadeIn(self.button))
         scene.play(Write(self.secret_key))
         scene.play(Write(self.address))
+
+    def generate_random_secret_key(self, scene, final_value: int):
+        rounds = 20
+        for i in range(rounds + 1):
+            random_key = randint(1, 40) if i < rounds else final_value
```
Comment (marekkirejczyk) [inline zkmarek/video/mobjects/wallet.py:49]: `randint(0, max_value)`

File: zkmarek/video/mobjects/wallet.py (hunk 1 above)
Comment (akonior) [reply zkmarek/video/mobjects/wallet.py]: done

File: zkmarek/video/mobjects/wallet.py [hunk 3]
```diff
@@ -34,3 +42,15 @@ def animate_in(self, scene):
         scene.play(FadeIn(self.border), FadeIn(self.clip), FadeIn(self.button))
         scene.play(Write(self.secret_key))
         scene.play(Write(self.address))
+
+    def generate_random_secret_key(self, scene, final_value: int):
```
[the remark this answers is not in the document]
Comment (akonior) [reply zkmarek/video/mobjects/wallet.py]: done

File: zkmarek/video/mobjects/wallet.py (hunk 2 above)
Comment (akonior) [reply zkmarek/video/mobjects/wallet.py]: done

**Review by marekkirejczyk — APPROVED** [review]

**Pull request merged**